src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -465,16 +480,23 @@ export default {
 	}
 
 	&[contenteditable='false'] {
+		border: 0;
+		opacity: 1;
+		padding: 0;
```

Comment:
Styling should be consistent. Removing padding makes it look entirely different.